doc/source/cookbook/geographic_projections.rst

```diff
@@ -0,0 +1,6 @@
+.. _cookbook-geographic_projections:
+
+Using Geographic Transforms and Projections
+-------------------------------------------
+
+.. notebook:: geographic_xforms_and_projections.ipynb
```

Comment:
I looked at the rendered `geographic_xforms_and_projections.ipynb` file and noticed that it still shows the default projection to the `PlateCarree` in cell 7.  Is that still the case?

Reply:
Yes it is PlateCarree for the geographic coordinate handler. Should I change it? Or do we want to leave PlateCarree for the geographic handler and set it to Mollweide for a different handler?

Reply:
Updated to Mollweide in b5c2d96e8016efa4d251755874cbe1e82a26f702